engine/src/main/java/com/datatorrent/stram/StramLocalCluster.java

```diff
@@ -72,7 +72,7 @@
   protected final StreamingContainerManager dnmgr;
   private final UmbilicalProtocolLocalImpl umbilical;
   private InetSocketAddress bufferServerAddress;
-  private boolean perContainerBufferServer;
+  private boolean perContainerBufferServer = true;
```

Comment:
Same question as well, I think we should limit the changes to what is really needed.